package.json

```diff
@@ -2,6 +2,7 @@
   "name": "gridicons",
   "version": "0.1.0",
   "devDependencies": {
+    "classnames": "^2.2.3",
```

Comment:
Careful here. This would break in a prod install. I would recommend removing the classnames dep entirely.